docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer), navigate to the **Developer** dropdown and select the **Chain state** option
+
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-1.webp)
+
+2. Under the **configuration** section, call the `activeConfig()` function by clicking the button with the **+** icon. This will display the current settings for all the Polkadot network parameters, including the HRMP channel settings
+
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-2.webp)
+
+### Dispatching Extrinsics
+
+Establishing new HRMP channels between parachains requires dispatching specific extrinsic calls on the Polkadot relay chain from the parachain's origin.
+
+The most straightforward approach is to implement the channel opening logic off-chain, then use the XCM pallet's `send` extrinsic to submit the necessary instructions to the relay chain. However, the ability to send arbitrary programs through the `Transact` instruction in XCM is typically restricted to privileged origins, such as the sudo pallet or governance mechanisms.
+
+Parachain developers have a few options for triggering the required extrinsic calls from their parachain's origin, depending on the configuration and access controls defined:
+
+- Sudo - if the parachain has a sudo pallet configured, the sudo key holder can use the sudo extrinsic to dispatch the necessary channel opening calls
+- Governance - the parachain's governance system, such as a council or OpenGov, can be used to authorize the channel opening calls
+- Privileged Accounts - the parachain may have other designated privileged accounts that are allowed to dispatch the HRMP channel opening extrinsics
+
+## Opening HRMP Channels Between Parachains
+
+For establishing communication channels between parachains on the Polkadot network, using the HRMP protocol, the following steps are required:
+
+1. Channel request - the parachain that wants to open an HRMP channel must make a request to the parachain it wishes to have an open channel with
+2. Channel acceptance - the other parachain must then accept this request in order to complete the channel establishment
+
+This process results in a unidirectional HRMP channel, where messages can flow in only one direction between the two parachains.
+
+To enable bidirectional communication, an additional HRMP channel must be established in the opposite direction. This requires repeating the request and acceptance process, but with the parachains reversing their roles.
+
+Once both unidirectional channels are established, the parachains can then send messages back and forth freely through the bidirectional HRMP communication channel.
+
+### Procedure for Initiating HRMP Channel Setup
+
+This example will demonstrate how to open a channel between parachain 2500 and parachain 2600, using Rococo Local as the relay chain.
```

Comment:
Maybe to make the tutorial even more generic, you can say something like: 

`between the dispatcher (parachain 2500 in this tutorial) and the receiver (parachain 2600)`

For the rest of the tutorial, we can refer to them as dispatcher and receiver directly

For sure, dispatcher and receiver are the first words that came into my mind, but you understand my point.